api/docs/http_report.md

```diff
@@ -21,30 +21,30 @@ PUT
 
 ### Supported Format(s)
 
-PSON
+`application/json`, `text/pson`
 
 ### Parameters
 
 None
 
 ### Content
 
-The content of a report is typically generated by the Puppet Runtime and consists of a pson-serialization of `Puppet::Transaction::Report` object which in turn contains a structure of objects with of the following runtime types:
+The content of a report is typically generated by the Puppet Runtime and consists of a json-serialization of `Puppet::Transaction::Report` object which in turn contains a structure of objects with of the following runtime types:
 
 * `Puppet::Util::Log`
 * `Puppet::Util::Metric`
 * `Puppet::Resource::Status`
 * `Puppet::Transaction::Event`
 
-This pson-serialization is compliant with the endpoint's report jsonschema.
+This json-serialization is compliant with the endpoint's report json schema.
```

Comment:
change to:
`This JSON serialization is compliant with the endpoint's report JSON schema.`